code/include/object_tracker.h

```diff
@@ -32,8 +37,31 @@ namespace picopter {
             void SetTrackMethod(TrackMethod method);
             void Run(FlightController *fc, void *opts) override;
             bool Finished() override;
+
+            cv::Vec3d GroundFromGPS(navigation::Coord3D coord);
+            navigation::Coord3D GPSFromGround(cv::Vec3d coord);
+
+            navigation::Coord3D launch_point;       //centre and orientation of the ground coordinate system
```

Comment:
This needs to be set in the object tracker Run function - it's currently not set anywhere which is causing it to have undefined values.

Also ideally this should not be public. If you need to access it from outside, I suggest a getter/setter function.